packages/core/src/demo/utils/stackblitz/angular.ts

```diff
@@ -71,98 +93,111 @@ bootstrapApplication(App)
   `
 
 	const angularJson = `{
-  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
-  "version": 1,
-  "newProjectRoot": "projects",
-  "projects": {
-    "demo": {
-      "root": "",
-      "sourceRoot": "src",
-      "projectType": "application",
-      "prefix": "app",
-      "schematics": {},
-      "architect": {
-        "build": {
-          "builder": "@angular-devkit/build-angular:browser-esbuild",
-          "options": {
-            "outputPath": "dist",
-            "index": "src/index.html",
-            "main": "src/main.ts",
-            "assets": [],
-            "styles": ["src/styles.css"],
-            "scripts": []
-          },
-          "configurations": {
-            "production": {
-              "fileReplacements": [
-                {
-                  "replace": "src/environments/environment.ts",
-                  "with": "src/environments/environment.prod.ts"
-                }
-              ],
-              "optimization": true,
-              "outputHashing": "all",
-              "sourceMap": false,
-              "extractCss": true,
-              "namedChunks": false,
-              "aot": true,
-              "extractLicenses": true,
-              "vendorChunk": false,
-              "buildOptimizer": true
+    "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+    "cli": {
+      "analytics": "1e1de97b-a744-405a-8b5a-0397bb3d01ce"
```

Comment:
what's this? do we need it? can we remove it?

Reply:
It's part of StackBlitz's Angular TypeScript template. The problem they are trying to solve is to prevent this prompt from coming up and stopping the execution...
`Would you like to share anonymous usage data with the Angular Team at Google under
Google’s Privacy Policy at https://policies.google.com/privacy? For more details  and how to change this setting, see http://angular.io/analytics. (y/N)`

I changed it to false and this appears to suppress the Angular prompt.